src/node.rs

```diff
@@ -9,27 +12,39 @@ use crate::sys::{new_map_with_capacity, ChildrenVec, Map, Vec};
 use crate::Error;
 use core::sync::atomic::{AtomicUsize, Ordering};
 
+/// A function that can be applied to a `Size<Number>` to obetain a `<Size<f32>`
 pub enum MeasureFunc {
+    /// Stores a unboxed function
     Raw(fn(Size<Number>) -> Size<f32>),
+    /// Stores a boxed function
     #[cfg(any(feature = "std", feature = "alloc"))]
     Boxed(Box<dyn Fn(Size<Number>) -> Size<f32>>),
 }
 
 /// Global sprawl instance id allocator.
 static INSTANCE_ALLOCATOR: Allocator = Allocator::new();
 
+/// An [`Id`]-containing identifier
 #[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
 #[cfg_attr(not(any(feature = "std", feature = "alloc")), derive(hash32_derive::Hash32))]
 pub struct Node {
+    /// The id of the forest that this node lives with
     instance: Id,
+    /// The identifier of this particular node within the tree
     local: Id,
 }
 
+/// A forest of UI [`Nodes`](`Node`), suitable for UI layout
 pub struct Sprawl {
+    /// The ID of the root node
     id: Id,
-    nodes: Allocator,
+    /// A monotonically-increasing index that tracks the [`Id`] of the next node
+    allocator: Allocator,
```

Comment:
This is also a breaking change for the changelog

Reply:
No, this field is private.